services/core-api/app/api/utils/access_decorators.py

```diff
@@ -32,15 +35,15 @@
 
 
 def is_minespace_user():
-    return jwt.validate_roles([MINESPACE_PROPONENT])
+    return getJwtManager().validate_roles([MINESPACE_PROPONENT])
```

Comment:
going forward all JWT use should go through `getJwtManager()` which will pick the right jwt object based on the token in header.